config/twill.php

```diff
@@ -217,6 +217,14 @@
     */
     'auto_seed_singletons' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Revisions configuration
+    |--------------------------------------------------------------------------
+    */
+    'max_revisions' => false,
```

Comment:
I am wondering if we can change this.

We could use:

```php
twill.revisions.global_limit = null|int;
twill.revisions.module_limit = [
  'module' => 'limit'
];
```

Where the more specified always takes over?